src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
+using Microsoft.CodeAnalysis.Collections;
```

Comment:
Will do. Makes me wonder if we could fix the "add using" fixer to insert differently in the first place.